metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/LinkCheckingPlugin.java

```diff
@@ -0,0 +1,45 @@
+package eu.europeana.metis.core.workflow.plugins;
+
+import eu.europeana.cloud.service.dps.DpsTask;
+import java.util.Map;
+
+/**
+ * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
+ * @since 2018-05-16
+ */
+public class LinkCheckingPlugin extends AbstractMetisPlugin {
```

Comment:
This class is now practically identical to the MediaProcessPlugin. Maybe we can merge these two classes?